code/modules/mob/living/carbon/life.dm

```diff
@@ -55,6 +55,7 @@
 	if(knocked_out || sleeping || getOxyLoss() > CARBON_KO_OXYLOSS || health < get_crit_threshold())
 		if(stat != UNCONSCIOUS)
 			blind_eyes(1)
+			DEAF(1<<2)
 		stat = UNCONSCIOUS
 	else if(stat == UNCONSCIOUS)
 		stat = CONSCIOUS
```

Comment:
Addtionally don't forget to remove the deafness when they are no longer unconscious

Reply:
Don't forget this